src/components/HomeCards/prices.tsx

```diff
@@ -1,155 +1,31 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import styled from 'styled-components'
-import { useSpring, animated } from 'react-spring'
+import numeral from 'numeral'
```

Comment:
How this is used?

Reply:
it's used for formatting numbers, so instead of 1000000 it can format to 1M